src/main/java/com/spotinst/sdkjava/model/bl/ocean/kubernetes/K8sVirtualNodeGroupCreationRequest.java

```diff
@@ -0,0 +1,60 @@
+package com.spotinst.sdkjava.model.bl.ocean.kubernetes;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.spotinst.sdkjava.client.rest.JsonMapper;
+import com.spotinst.sdkjava.model.OceanK8sVirtalNodeGroupConverter;
+import com.spotinst.sdkjava.model.api.ocean.kubernetes.ApiK8sVirtualNodeGroup;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class K8sVirtualNodeGroupCreationRequest {
```

Comment:
Is this class needed?

Reply:
Yes, this class is needed.  I will combine all the request classes into one request class instead of three

Reply:
I will rename to use the same Vng naming